src/utils/domain.ts

```diff
@@ -62,3 +62,14 @@ export const getAvatarPath = (avatar?: string, placeholder?: string) => {
 
   return `${STORAGE_URL}/${avatar}`;
 };
+
+export const getAvatarPathWithInitialsFallback = (
```

Comment:
how about `getAvatarPathWithFallback` for brevity's sake? also a bit more future-proof if we change the implementation

Reply:
Yeah, sounds good to me.